apps/lib/apps.c

```diff
@@ -942,13 +945,14 @@ int load_key_certs_crls(const char *uri, int format, int maybe_stdin,
     OSSL_PARAM itp[2];
     const OSSL_PARAM *params = NULL;
 
+    /* 'failed' describes type of credential to load for potential error msg */
     if (failed == NULL) {
         if (!quiet)
-            BIO_printf(bio_err, "Internal error: nothing to load from %s\n",
+            BIO_printf(bio_err, "Internal error: nothing was requested to load from %s\n",
                        uri != NULL ? uri : "<stdin>");
         return 0;
     }
-    ERR_set_mark();
+    ERR_set_mark(); /* suppress (most) low-level errors during loading */
```

Comment:
Is this really about *low level* errors? Isn't it instead about not polluting the error stack with extraneous messages, due from recoverable/ignorable conditions? Perhaps "low-level" is not the best choice of words here...

Reply:
Since this PR is of 8+ months back, I do not recall details on the error messages I observed there.
Would have been much more efficient to receive comments soon after PR submission.

Yes, what I meant with "low-level errors" is mostly extraneous error queue entries,
which should have been removed by the library on recovering from failed parse attempts.
Changed to `suppress any extra errors left over from failed parse attempts`.